backend/routes/bounty.js

```diff
@@ -0,0 +1,59 @@
+const express = require('express');
+const router = express.Router();
+const validateBountyInput = require('../validation/bounty');
+
+const Bounty = require('../models/Bounty');
+
+router.post('/bounty', function(req, res) {
+    const { errors, isValid } = validateBountyInput(req.body);
+
+    if(!isValid) {
+        return res.status(400).json(errors);
+    }
+
+    Bounty.findOne({
+        title: req.body.title
+    }).then(bounty => {
+        if(bounty){
```

Comment:
please put a space between the ) and { the rest of the code looks like we are having a space there